.github/workflows/wheel_linux_x86_64.yml

```diff
@@ -155,7 +155,7 @@ jobs:
             source /opt/rh/devtoolset-11/enable -y
 
           CIBW_ENVIRONMENT: |
-            PATH=/opt/rh/devtoolset-11/root/usr/bin:$PATH CMAKE_ARGS="-DPYTHON_EXECUTABLE=$(which python) -DKokkos_ENABLE_${{ matrix.exec_model }}=ON"
+            PATH=/opt/rh/devtoolset-11/root/usr/bin:$PATH CMAKE_ARGS="-DPYTHON_EXECUTABLE=$(which python)"
```

Comment:
Do we no longer need to specify which backend is being built here?

Reply:
It is not needed as this is specified when building the Kokkos libs, which is now done separately.

Reply:
Regarding that, we are still building Kokkos on-the-fly in the test workflows. I would like to get rid of that in another PR. Just have one workflow to build and cache Kokkos-BACKEND, then reuse the libs in all workflows that need it.

Reply:
Sounds good. For the suggested change we can still build the backend of choice with setup.py though right?

Reply:
Yes, this is done as before passing `-DKokkos_ENABLE_BACKEND=ON` in `CMAKE_ARGS`.

Reply:
Actually, this is still done in the GPU tests, thus proving it works.

Reply:
All good so. Thanks @vincentmr 
I'll approve.